src/doc/rustc/src/codegen-options/index.md

```diff
@@ -71,9 +71,11 @@ If not specified, debug assertions are automatically enabled only if the
 This flag controls the generation of debug information. It takes one of the
 following values:
 
-* `0`: no debug info at all (the default).
-* `1`: line tables only.
-* `2`: full debug info.
+* `0` or `none`: no debug info at all (the default).
+* `line-directives-only`: line info directives only.
+* `line-tables-only`: line tables only.
+* `1` or `limited`: debug info without type information.
+* `2` or `full`: full debug info.
```

Comment:
It would be good to document what all these options actually mean, in practice.
I.e. in which ways can you debug an executable build with the given option.

Also, it would be good to define what "line info directives" and "line tables" mean, at least by giving links to relevant documents.

Reply:
Is the new documentation I added clear enough for unfamiliar users?